src/containers/pages/FocusInvestigation/single/tests/index.test.tsx

```diff
@@ -145,4 +145,64 @@ describe('containers/pages/SingleFI', () => {
     expect(wrapper.find('GisidaWrapperMock').props()).toMatchSnapshot();
     wrapper.unmount();
   });
+
+  it('calls superset with the correct params', async () => {
+    const getFormDataMock: any = jest.fn();
+    getFormDataMock.mockImplementation(() => {
+      return {};
+    });
+    superset.getFormData = getFormDataMock;
+    const mock: any = jest.fn();
+    const supersetMock: any = jest.fn();
+    supersetMock.mockImplementation(() => Promise.resolve([]));
+    store.dispatch(fetchPlans(fixtures.plans as Plan[]));
+    store.dispatch(fetchGoals(fixtures.goals));
+    store.dispatch(fetchJurisdictions(fixtures.jurisdictions));
+    const props = {
+      history,
+      jurisdiction: fixtures.jurisdictions[0],
+      location: mock,
+      match: {
+        isExact: true,
+        params: { id: fixtures.plan1.id },
+        path: `${FI_SINGLE_URL}/:id`,
+        url: `${FI_SINGLE_URL}/16`,
+      },
+      supersetService: supersetMock,
+    };
+    const wrapper = mount(
+      <Provider store={store}>
+        <Router history={history}>
+          <ConnectedSingleFI {...props} />
+        </Router>
+      </Provider>
+    );
+    // HACK: HACK: Force async code in componentDidMount to resolve
+    await Promise.resolve();
+    await Promise.resolve();
+    await Promise.resolve();
+    const getFormDataCallList = [
+      [3000, [{ comparator: fixtures.plan1.plan_id, operator: '==', subject: 'plan_id' }]],
+      [
+        3000,
+        [{ comparator: fixtures.plan1.plan_id, operator: '==', subject: 'plan_id' }],
+        { action_prefix: true },
+      ],
+      [
+        3000,
+        [
+          {
+            comparator: fixtures.plan1.jurisdiction_id,
+            operator: '==',
+            subject: 'jurisdiction_id',
+          },
+        ],
+      ],
+    ];
+    expect((superset.getFormData as any).mock.calls).toEqual(getFormDataCallList);
+    const callList = [[0, {}], [3, {}], [1, {}]];
```

Comment:
Is there no way to make sure we dont get empty objects here?  I think it makes the test very confusing